libs/base/Data/List.idr

```diff
@@ -620,3 +622,12 @@ revAppend (v :: vs) ns
           rewrite sym (revAppend vs ns) in
             rewrite appendAssociative (reverse ns) (reverse vs) [v] in
               Refl
+
+export
+dropSum : (n, m : Nat) -> (l : List t) -> drop n (drop m l) = drop (n+m) l
```

Comment:
I would call this `dropFusion` just like we can have `mapFusion`
(I looked for such a lemma but could not find it, strangely).